stable/dask-distributed/templates/dask-jupyter-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "jupyter-fullname" . }}-config
+  labels:
+    app: {{ template "name" . }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Release.Name }}-{{ .Values.jupyter.component }}"
+data:
+  jupyter_notebook_config.py: |
+    c = get_config()
+    c.NotebookApp.password = '{{ .Values.jupyter.password }}'
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "jupyter-fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Release.Name }}-{{ .Values.jupyter.component }}"
+spec:
+  replicas: {{ .Values.jupyter.replicas }}
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      component: "{{ .Release.Name }}-{{ .Values.jupyter.component }}"
```

Comment:
Add `app` and `release` label to selector. You may as well remove the selector completely.